ow_dynamic_terrain/src/DynamicTerrainModel.cpp

```diff
@@ -34,6 +34,12 @@ class DynamicTerrainModel : public ModelPlugin, public DynamicTerrainBase
 
     m_model = model;
 
+    auto world = model->GetWorld();
+    if (world) {
+      world->Physics()->SetParam("ode_quiet", true);
+      gzerr << m_plugin_name << ": ODE's LCP Error messages have been suppressed!" << endl;
+    }
```

Comment:
I guess SDFormat hasn't been updated yet to include this as a parameter for .world files?

Just to be sure it wasn't just omitted documentation, I tried adding a <ode_quiet>true</ode_quiet> line to each level below physics in the a world file, but it never took. So I can only assume a function call is the only way to set this parameter.